### ISSUE TITLE
example code broken

### ISSUE THREAD

**Issue by gkiko** [websockets/ws#731]
```
OS X 10.11.5
node v6.2.0
```

`npm install` downloads **express@4.13.4** package in _fileapi_ example. Executing `node server.js` throws following exception

```
/Users/gkiko/Workspace/ws/examples/serverstats/server.js:4
  , app = express.createServer();
                  ^

TypeError: express.createServer is not a function
    at Object.<anonymous> (/Users/gkiko/Workspace/ws/examples/serverstats/server.js:4:19)
```
#### fix

downgrade to **express@2.x**

**Comment by JoshuaWise** [comment]
pull requests are welcome!

**Comment by eliezergu** [comment]
it looks like npm won't install express 2.x.
so what's now???

**Comment by lpinca** [comment]
@eliezergu fixing the example to use `express@4` shouldn't be hard.
Would you mind sending a pull request?

**Comment by lpinca** [comment]
Fixed by #953, closing.
